tests/Twilio/Unit/Rest/ClientTest.php

```diff
@@ -241,4 +242,21 @@ public function testAuthorizationHeaderRemoval(): void {
         $this->assertStringNotContainsString('Authorization-header', stream_get_contents($capturedLogging));
     }
 
+    public function testDefaultUserAgent(): void {
+        $client = new Client('username', 'password');
+        $client->request('GET', 'https://api.twilio.com');
+        $userAgent = $client->getHttpClient()->{'lastRequest'}[CURLOPT_HTTPHEADER][0];
+        $expectedString = 'twilio-php/' . VersionInfo::string() . ' (' . php_uname("s") . ' ' . php_uname("m") . ')' . ' PHP/' . PHP_VERSION;
```

Comment:
Just a thought for future work; not sure if it's something to address in this change:

We should be careful about test code implementation copying source code implementation. For example, this line looks the same as the source implementation so it's pretty self-fulfilling. A way to test around this is to make the assertions a bit looser and assert on the format or that certain things are contained in the string. Some things that could be done would verify that the string begins with `twilio-php`, contains the helper version, contains the PHP version, etc.